### PR TITLE
Revert "Add latest MySQL 5.5/5.6/5.7 versions and macOS Sierra compatibility"

### PR DESCRIPTION
**Pull request by noam-almog**
It's breaking our linux builds, will look into it later on

### CODE REVIEW

File: src/main/java/com/wix/mysql/distribution/Version.java [hunk 1]
```diff
@@ -1,94 +1,49 @@
...
-        }
-    }
+    v5_5_40("5.5", "40", Platform.Linux, Platform.OS_X),
+    v5_6_21("5.6", "21"),
+    v5_6_22("5.6", "22"),
+    v5_6_23("5.6", "23"),
+    v5_6_24("5.6", "24"),
+    v5_6_latest(v5_6_24),
+    v5_7_10("5.7", "10"),
+    v5_7_13("5.7", "10"),
```
Comment (viliusl) [inline src/main/java/com/wix/mysql/distribution/Version.java]: @noam-almog - should not it be "13" for a minor?

Also we could expand os x build matrix - no sierra support yet, but better than current set-up: https://docs.travis-ci.com/user/osx-ci-environment/#OS-X-Version

Comment (noam-almog) [inline src/main/java/com/wix/mysql/distribution/Version.java]: @viliusl I just upgraded to sierra and my build passed without this change so no urgency in re-merging this. I'll dig into it tomorrow.

Comment (sparhomenko) [inline src/main/java/com/wix/mysql/distribution/Version.java]: Interestingly enough, the build consistently hangs on at least two Sierra laptops I tried it on. I guess the MySQL bug is only reproducible under certain conditions, but no idea what they might be.

**Comment by sparhomenko** [comment]
Build failure is specific to 5.7.11 where Oracle added `keyring_file` plugin and enabled it by default, a decision they [backtracked on](https://dev.mysql.com/doc/refman/5.7/en/keyring-installation.html):
>In MySQL 5.7.11, the default --early-plugin-load value was the name of the keyring_file plugin library file, so that plugin was loaded by default.
>As of MySQL 5.7.12, the default --early-plugin-load value is empty. To load a keyring plugin, you must use an explicit --early-plugin-load option with a nonempty value.

It can be fixed by adding `lib/plugin/keyring_file.so` to the list of files to extract, and creating empty data directory required by that plugin. However, it might be more practical to just drop 5.7.11 support: teams that are using 5.7 are likely to already be using a later build.

**Comment by noam-almog** [comment]
@sparhomenko can you fix it in the original PR ?

**Comment by sparhomenko** [comment]
@noam-almog Don't think Github allows reopening PRs after merging, created a new one: #70.

**Pull request merged**